src/components/UsersTableView.module.scss

```diff
@@ -0,0 +1,66 @@
+@import '../styles/common.scss';
+.userTableContainer {
+  width: 100%;
+  overflow-x: auto;
+  color: var(--semantic-fg-primary);
+  display: flex;
+  flex-direction: column;
+
+  .userTable {
+    border-collapse: collapse;
+    table-layout: fixed;
+    width: 100%;
+
+    th, td {
+      width: 308px;
```

Comment:
can you make the width non-constant, so that it will be dependent on the screen size? and also specify min width in px)